<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="imageDisplay">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1501</width>
      <height>801</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1478</width>
       <height>778</height>
      </rect>
     </property>
     <widget class="QLabel" name="imgHolder">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1471</width>
        <height>771</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="outputDisplay">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>10</y>
      <width>400</width>
      <height>460</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="changeImage">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>810</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startPointCount">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>810</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Start </string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="createGrid">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>810</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Grid</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageSize">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>815</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
